DotNet/parser.js

```diff
@@ -146,6 +154,10 @@
 		})
 		return result
 	}
+	if ((elements[2] !== undefined) && (elements[2].includes("batchInfo"))) {
+		if (elements[4].includes("length")) elements[4] = "Count()"
+		if (elements[4].includes('0')) elements[4] = "First()"
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
		if (elements[4].includes('0')) {elements[4] = "First()"}
```